SteamBot/Bot.cs

```diff
@@ -860,6 +860,19 @@ public void UnsubscribeTrade (UserHandler handler, Trade trade)
             trade.OnUserAccept -= handler.OnTradeAcceptHandler;
         }
 
+        /// <summary>
+        /// Fetch the Bot's inventory and log a warning if it's private
+        /// </summary>
+        private Inventory FetchBotsInventory()
+        {
+            var inventory = Inventory.FetchInventory(SteamUser.SteamID, ApiKey, SteamWeb);
+            if(inventory.IsPrivate)
+            {
+                log.Warn("The bot's backpack is private! If your bot adds any items it will fail! Your bot's backpack should be Friends Only or Public.");
```

Comment:
Oh, right, that `IEconItems_440/GetPlayerItems/` stuff.  I keep forgetting about that because I don't use it.

Yeah I think that will only support public inventories.  I'll change the message.